src/AnimatedRegion.ts

```diff
@@ -82,27 +95,13 @@ export default class AnimatedMapRegion extends AnimatedWithChildren {
 
   private __getValue() {
     return {
-      latitude: this.latitude.__getValue(),
-      longitude: this.longitude.__getValue(),
-      latitudeDelta: this.latitudeDelta.__getValue(),
-      longitudeDelta: this.longitudeDelta.__getValue(),
+      latitude: (this.latitude as any)._value,
+      longitude: (this.longitude as any)._value,
+      latitudeDelta: (this.latitudeDelta as any)._value,
+      longitudeDelta: (this.longitudeDelta as any)._value,
     };
   }
 
-  private __attach() {
-    this.latitude.__addChild(this);
-    this.longitude.__addChild(this);
-    this.latitudeDelta.__addChild(this);
-    this.longitudeDelta.__addChild(this);
-  }
-
-  private __detach() {
-    this.latitude.__removeChild(this);
-    this.longitude.__removeChild(this);
-    this.latitudeDelta.__removeChild(this);
-    this.longitudeDelta.__removeChild(this);
-  }
-
```

Comment:
those functions were unused and Animated.Value doesn't methods like __addChild or __removeChild
ref. https://reactnative.dev/docs/animatedvalue